scripts/add_gen3_repo.py

```diff
@@ -40,6 +42,10 @@ def __init__(self, **kwargs):
 
 
 def main():
+    # Ensure logs from tasks are visible
+    logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+    lsst.log.configure_pylog_MDC("DEBUG", MDC_class=None)
+
     args = _Parser().parse_args()
     log = lsst.log.Log.getLogger("add_gen3_repo")
```

Comment:
Can you switch this to python logging since you are enabling it above? There is also a log.warn below that should be a log.warning.

Reply:
How soon are `lsst.log` and `.warn` going away? This script will become obsolete once Gen 2 is removed, if not sooner, so I'm not in a hurry to clean up deprecations.

Reply:
`lsst.log` is not going away since it needs to work for C++ code. What we are doing is removing all of it from Python code. If this script is only going to last a short while until gen2 is removed then it's not important to make the two changes I requested.